v2/emailpassword/common-customizations/styling/changing-style.mdx

```diff
@@ -66,6 +66,29 @@ The above will result in:
 
 <img width="500px" src="/img/emailpassword/signup-centered-blue-button.png" />
 
+### Customizing common components
```

Comment:
- The title is not clear - perhaps call it "Global style changes"?
- The placement of this section is not good - It should probably be it's own "##" section at the top of this file (as the first section), and then all the other sections will be specific to each form.

(Same comment applies to other recipes as well)